src/gui/plugins/transform_control/TransformControl.qml

```diff
@@ -339,153 +339,40 @@ ToolBar {
           Layout.column: 0
           bottomPadding: 10
         }
+
         Text {
-          text: "X"
-          color: snapItem
-          Layout.row: 1
-          Layout.column: 0
-        }
-        IgnSpinBox {
-          id: xEntry
-          minimumValue: 0.01
-          maximumValue: 100.0
-          decimals: 2
-          stepSize: 0.01
-          value: 1
-          Layout.row: 1
-          Layout.column: 1
-          onEditingFinished: {
-            TransformControl.OnSnapUpdate(
-              xEntry.value, yEntry.value, zEntry.value,
-              rollEntry.value, pitchEntry.value, yawEntry.value,
-              0, 0, 0
-            )
-          }
-        }
-        Text {
-          text: "Y"
-          color: snapItem
-          Layout.row: 2
-          Layout.column: 0
-        }
-        IgnSpinBox {
-          id: yEntry
-          minimumValue: 0.01
-          maximumValue: 100.0
-          decimals: 2
-          stepSize: 0.01
-          value: 1
-          Layout.row: 2
-          Layout.column: 1
-          onEditingFinished: {
-            TransformControl.OnSnapUpdate(
-              xEntry.value, yEntry.value, zEntry.value,
-              rollEntry.value, pitchEntry.value, yawEntry.value,
-              0, 0, 0
-            )
-          }
-        }
-        Text {
-          text: "Z"
-          color: snapItem
-          Layout.row: 3
-          Layout.column: 0
-        }
-        IgnSpinBox {
-          id: zEntry
-          minimumValue: 0.01
-          maximumValue: 100.0
-          decimals: 2
-          stepSize: 0.01
-          value: 1
-          Layout.row: 3
-          Layout.column: 1
-          onEditingFinished: {
-            TransformControl.OnSnapUpdate(
-              xEntry.value, yEntry.value, zEntry.value,
-              rollEntry.value, pitchEntry.value, yawEntry.value,
-              0, 0, 0
-            )
-          }
-        }
-        Text {
-          text: "Rotation (deg)"
+          text: "Rotation (rad)"
```

Comment:
This is not working correctly since `TransformControl.OnSnapUpdate` takes rpy in as degrees: https://github.com/gazebosim/gz-sim/blob/6d1582f8371f064e1aa0c461fc9031ae6c046363/src/gui/plugins/transform_control/TransformControl.hh#L64-L69

Also, we should update it so that it shows as degrees since its easier to read/write.